recipes/fire-hpp/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.2.0-pre.20201031":
```

Comment:
Can't we just wait on a new release?
Release `0.1` was done on July 3th: https://github.com/kongaskristjan/fire-hpp/releases

Reply:
I explicitly asked the author to make a new release but they don't want to yet, and 0.1 isn't worth packaging because the API has already been broken and would require its own bespoke recipe. I'm sorry if the release number isn't pretty enough for you.

Reply:
Understood, so please use the version format `cci.20201031`: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
That is not the same situation. The library has official releases and it's not like RapidJSON which has no roadmap. The next release will be 0.2.0, so shouldn't this be a pre-release?

Reply:
It is an established standard in CCI to use the `cci.<date-of-commit>` format for all versions we create ourselves - no matter what the underlying motivation is.

The documentation needs to be updated on this however. Thanks for the hint.

Reply:
Author of fire-hpp here. I also strongly suggest waiting for v0.2. For the last two days I've had much more time and everything is now almost ready for release. I'll try to do the release tomorrow, or if I can, even perhaps today.

Reply:
Thank you @kongaskristjan

Reply:
Okay then, I'll just close this then and re-open when 0.2 is out. It'll give me the chance to squash the branch commits. Sorry for getting a little bitter, I still like you all. :)

Reply:
Well that was fast! Replaced with true-blue official 0.2. :)